content/22.vaccines.md

```diff
@@ -304,22 +304,68 @@ Limited distribution was shared with the UAE [@url:https://www.nytimes.com/inter
-<!--To Do: Application/advancements in prior 21st century epidemics?-->
-
-NVX-CoV2373, produced by U.S. company Novavax, is a protein nanoparticle vaccine candidate against SARS-CoV-2 that is constructed from a mutated prefusion SARS-CoV-2 spike protein in combination with a specialized adjuvant to elicit an immune response against SARS-CoV-2.
-The spike protein is recombinantly expressed in Sf9 insect cells [@doi:10.1038/s41467-020-20653-8], which have previously been used for several other FDA-approved protein therapeutics [@doi:10.1021/acs.iecr.8b00985] and contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) along with two proline substitutions (K986P and V987P) that improve thermostability [@doi:10.1038/s41467-020-20653-8].
-In preclinical mouse models, Novavax-CoV2373 elicited high anti-spike IgG titers 21 to 28 days post-vaccination that could neutralize the SARS-CoV-2 virus and protect the animals against virus challenge, with particularly strong effects when administered with the proprietary adjuvant Matrix-M^TM^ [@doi:10.1038/s41467-020-20653-8].
-In a phase I/II trial, a two-dose regimen of NVX-CoV2373 was found to induce anti-spike IgG levels and neutralizing antibody-titers exceeding those observed in convalescent plasma donated by symptomatic patients [@doi:10.1056/NEJMoa2026920].
-In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
-Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase III trial in the UK and also noted that 90% of cases occurring in their phase IIb study in South Africa were caused by a variant of concern, B.1.351 [@url:https://ir.novavax.com/news-releases/news-release-details/novavax-covid-19-vaccine-demonstrates-893-efficacy-uk-phase-3].
-Despite these very preliminary results, Novavax has signed an agreement with the Serum Institute of India allowing them to produce up to 2 billion doses a year [@url:https://www.reuters.com/article/health-coronavirus-novavax-idUSKBN2661PI] and has also signed agreements with the U.K., Canada, Australia, and South Korea [@url:https://ir.novavax.com/news-releases/news-release-details/novavax-announces-expanded-collaboration-and-license-agreement] as well as projecting that they will supply 1.1 billion doses to COVAX for distribution to countries with limited access to vaccine supplies [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
-<!--To Do: closing sentence contextualizing role protein subunit vaccines may play in this pandemic relative to previous ones-->
+This platform is also favoured for its consistency in production and defined components designed for a highly targeted immune response to a specific pathogen using synthetic immunogenic particles that can be designed to avoid allergen and reactogenic sequences [@doi:10.3390/vaccines2030515].
```

Comment:
```suggestion
This platform is also favored for its consistency in production and defined components designed for a highly targeted immune response to a specific pathogen using synthetic immunogenic particles that can be designed to avoid allergen and reactogenic sequences [@doi:10.3390/vaccines2030515].
```